### PR TITLE
Remove overly specific default cli options

### PR DESCRIPTION
**Pull request by sphuber**
Also renamed the 'parent_calc' option to 'calculation' which
is clearer and more accurate

### CODE REVIEW

File: aiida/utils/cli/validators.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def validate_kpoint_mesh(callback_kwargs, ctx, param, value):
 
     return kpoints
 
-def validate_parent_calc(callback_kwargs, ctx, param, value):
+def validate_validate_calculation(callback_kwargs, ctx, param, value):
```
Comment (ltalirz) [inline aiida/utils/cli/validators.py]: that should be `validate_calculation`

File: aiida/utils/cli/options.py [hunk 2]
```diff
@@ -44,60 +44,59 @@ def __call__(self, *args, **kwargs):
...
     help='the number of points in the kpoint mesh along each basis vector'
 )
 
-parent_calc = overridable_option(
-    '-r', '--parent-calc', type=click.INT, required=True,
-    callback=validators.validate_parent_calc,
-    help='the node pk of the parent calculation'
+
+calculation = overridable_option(
+    '-r', '--calculation', type=click.INT, required=True,
```
Comment (ltalirz) [inline aiida/utils/cli/options.py]: what about using uppercase `-C`?
as it stands, it really isn't apparent why it would be `-r`

**Comment by ltalirz** [comment]
For me is ok, but you still have the typo `validate_validate_calculation`

**Comment by sphuber** [comment]
Ah shit, didn't see that first time around, will fix it now. Thanks!

**Pull request merged**